english/coffee/drink-tea/drink-hot-tea.md

```diff
@@ -1,3 +1,4 @@
 The hot tea instantly makes the chilli left in your mouth feel a lot hotter,
-but soon the burning sensation passes. 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.